README.md

```diff
@@ -1,6 +1,11 @@
-<p align="center">
-  <img src="assets/Sketchware-Pro.png" />
+<p>
+  <img src="assets/Sketchware-Pro.png"  alt="Sketchware Pro logo"/>
 </p>
+<style>
+p {
+    text-align: center;
+}
+</style>
```

Comment:
this breaks the centered image [even though the previous approach is the only one that works](https://stackoverflow.com/a/62383408)

Reply:
fixed